imports/ui/components/settings/Settings.jsx

```diff
@@ -293,10 +317,10 @@ class SettingsPage extends Component {
                                 <h5>personal information</h5>
                             </div>
                             <div className={theme.cardContent}>
-                                <h6>name: <span>shahid afridi</span></h6>
-                                <h6>contact number: <span>+92-300-32198765</span></h6>
-                                <h6>email: <span>shahidafridi@hotmail.com</span></h6>
-                                <h6>address: <span>house no.10, block j, karachi, pakistan</span></h6>
+                                <h6>name: <span>{Meteor.user().profile.fullName}</span></h6>
+                                <h6>Contact Number: <span> {Meteor.user().profile.contactNumber || 'Not Available'}</span></h6>
+                                <h6> <span> {Meteor.user().username ? 'Username:' : 'Email:' } </span> {Meteor.user().username ? Meteor.user().username : Meteor.user().emails[0].address }</h6>
+                                <h6>Address: <span> {Meteor.user().profile.address || 'Not Available'}</span></h6>
                                 <div className={theme.settingBtn}>
```

Comment:
We're storing these properties in `state`, so use `state` here, not `Meteor.users` again.